pkgs/darwin-uninstaller/configuration.nix

```
@@ -16,12 +16,12 @@ with lib;
   nix.useDaemon = mkForce false;
 
   system.activationScripts.postUserActivation.text = mkAfter ''
-    if [[ -L ~/.nix-defexpr/channels/darwin ]]; then
-        nix-channel --remove darwin || true
-    fi
+    nix-channel --remove darwin || true
```

```suggestion
    nix-channel --remove darwin
```

Surely not this one as well…